src/component/project/ProjectAccess/ProjectAccessTable/ProjectAccessTable.tsx

```diff
@@ -43,6 +43,11 @@ import { ProjectAccessCreate } from 'component/project/ProjectAccess/ProjectAcce
 import { ProjectAccessEditUser } from 'component/project/ProjectAccess/ProjectAccessEditUser/ProjectAccessEditUser';
 import { ProjectAccessEditGroup } from 'component/project/ProjectAccess/ProjectAccessEditGroup/ProjectAccessEditGroup';
 import useHiddenColumns from 'hooks/useHiddenColumns';
+import {
+    PA_ASSIGN_BUTTON_ID,
+    PA_EDIT_BUTTON_ID,
+    PA_REMOVE_BUTTON_ID,
+} from '../../../../utils/testIds';
```

Comment:
Should avoid relative paths.
```suggestion
} from 'utils/testIds';
```

Reply:
Damn the auto import.